mesmer/stats/_harmonic_model.py

```diff
@@ -267,7 +267,7 @@ def fit_harmonic_model(yearly_predictor, monthly_target, max_order=6, time_dim="
     data_vars : `xr.Dataset`
         Dataset containing the selected order of Fourier Series (selected_order),
         the estimated coefficients of the Fourier Series (coeffs) and the resulting
-        predictions for monthly values (predictions).
+        residuals of th model (residuals).
```

Comment:
```suggestion
        residuals of the model (`residuals`).
```

Enclose all variable names in backticks?